.ruff.toml

```
@@ -274,10 +286,12 @@ lint.unfixable = [
     "PTH",      # all flake8-use-pathlib
     "S321",  # Suspicious-ftp-lib-usage
     "TRY300",  # Consider `else` block
+    "B007",  # UnusedLoopControlVariable
 ]
 "astropy/visualization/*" = [
     "B015",
     "PLR0911",  # too-many-return-statements
+    "B007",  # UnusedLoopControlVariable
```

Keep all these lists alphabetically sorted (I'm only making an explicit suggestion for the `visualization` list):
```suggestion
    "B007",  # UnusedLoopControlVariable
    "B015",
    "PLR0911",  # too-many-return-statements
```

---

Keep all these lists alphabetically sorted, not just the one for `visualization`.

---

it should be all alphabetically sorted